### DISCUSSION TITLE
View multiple pieces of text as one

### DISCUSSION THREAD

**Discussion by xahodo** [vkbo/novelWriter#1375] [Ideas]
Scrivener has this "scrivenings" feature, which you turn on, select a few of the sections you want to see from the binder and it displays them as one huge piece of text, while still keeping them separate. Would such a feature also be possible for novelWriter?

This would allow one to see whether a certain scene could be missed/what happens to a story if a certain part or parts are left out, or added in. Bonus points if sections can be added, removed, or moved while in this mode. Of course changes in the order of sections and which sections are used in this mode would immediately be reflected in the display.

NOTE: this is *different* then the join feature as, for all intents and purposes, the texts are not *actually* merged into one piece of text.

I apologize if this post is a bit confusing, I'm having a hard time explaining it.

**Comment by vkbo** [comment]
It's already been suggested in #102. There will likely not be an option to edit multiple documents at once, due to several technical reasons, but in principle it should be possible to view multiple documents. The feature request is still open for that reason.
